apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/service/AccountStatisticsService.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright © 2018-2020 Apollo Foundation
+ */
+package com.apollocurrency.aplwallet.apl.core.rest.service;
+
+import com.apollocurrency.aplwallet.api.dto.account.AccountEffectiveBalanceDto;
+import com.apollocurrency.aplwallet.api.dto.account.AccountsCountDto;
+import com.apollocurrency.aplwallet.apl.core.account.model.Account;
+import com.apollocurrency.aplwallet.apl.core.account.service.AccountService;
+import com.apollocurrency.aplwallet.apl.core.app.Blockchain;
+import com.apollocurrency.aplwallet.apl.core.app.Convert2;
+import com.apollocurrency.aplwallet.apl.core.chainid.BlockchainConfig;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.util.List;
+import java.util.Objects;
+import java.util.Random;
+
+/**
+ * Account statistic information.
+ */
+@Slf4j
+@Singleton
+public class AccountStatisticsService {
+    private BlockchainConfig blockchainConfig;
+    private Blockchain blockchain;
+    private AccountService accountService;
+
+    @Inject
+    public AccountStatisticsService(BlockchainConfig blockchainConfig,
+                                    Blockchain blockchain,
+                                    AccountService accountService) {
+        this.blockchainConfig = Objects.requireNonNull(blockchainConfig, "blockchainConfig is NULL");
+        this.blockchain = Objects.requireNonNull(blockchain, "blockchain is NULL");
+        this.accountService = Objects.requireNonNull(accountService,"accountService is NULL");
+    }
+
+    public AccountsCountDto getAccountsStatistic(int numberOfAccounts) {
```

Comment:
I'd moved this method to account service but without conversion to dto. AccountService collect accountStatistic into entityClass, then AccountStatisticConverter make a dto of that entity by utilizing AccountConverter if needed.

Reply:
There is no table entity like 'AccountStatistic' because it's totally composed information taken from Account + using EffectiveBalance + GuaranteedBalance data.